src/earn/EarnEnterAmount.tsx

```diff
@@ -374,30 +379,35 @@ function EarnEnterAmount({ route }: Props) {
         <EarnDepositBottomSheet
           forwardedRef={reviewBottomSheetRef}
           preparedTransaction={prepareTransactionsResult}
-          amount={tokenAmount.toString()}
+          amount={tokenAmount}
           tokenId={token.tokenId}
+          apy={apy}
+          token={token}
         />
       )}
     </SafeAreaView>
   )
 }
 
-function EarnProceed({
+export function EarnApyAndAmount({
+  apy,
   tokenAmount,
-  localAmount,
+  localCurrencySymbol,
   token,
-  amountEnteredIn,
-  disabled,
-  onPressProceed,
-  onPressInfo,
-}: ProceedComponentProps) {
+}: {
+  apy: number | undefined
+  tokenAmount: BigNumber | null
+  localCurrencySymbol: LocalCurrencySymbol | null
+  token: TokenBalance
+}) {
   const { t } = useTranslation()
-  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
 
-  const asyncPoolInfo = useAavePoolInfo({ depositTokenId: token.tokenId })
+  const apyString = apy ? (apy * 100).toFixed(2) : '--'
+  const earnUpTo =
+    apy && tokenAmount?.gt(0) ? tokenAmount.multipliedBy(new BigNumber(apy)).toFormat(2) : '--'
 
   return (
-    <View style={styles.infoContainer}>
+    <>
       <View style={styles.line}>
         <Text style={styles.label}>{t('earnFlow.enterAmount.earnUpToLabel')}</Text>
```

Comment:
This copy differs slightly between the enter amount and deposit screen ("You could earn up to:" vs. "You'll earn up to"). Is there a reason for this difference or can they be the same?

Reply:
can you check with product on this?

Reply:
confirmed with John to have both be "You could earn..."